### ISSUE TITLE
Make sending data to monitor optional 

### ISSUE THREAD

**Issue by badb** [golemfactory/clay#876]
Previously there was one option in configuration - if it was switched on, Golem node sends information to monitor, otherwise not. 
Check which option was that, if it's still supported and blocks monitor communication everywhere.

**Comment by maaktweluit** [comment]
TL:DR;
Think its in client.py constructor as `use_monitor`, but could not find any configuration.
Would like to help fix this, but need some hints getting started.

Full comment:
After some digging in the code i think this issue is about:
```./golem/client.py:80:            use_monitor=True```

This boolean disables the monitor in that file, it seems to do that correct but i am no expert. When turning it off it still builds and runs, i do not see any notable difference.

After a global search it did not seem to affect other files, what other channels of communication have to be checked before the monitor is properly off?

All configuration to enable it seems lost, all places where `' Client('` is called do not use the use_monitor parameter.

Also, none of the versions until 0.4.0 mention anything in the --help about more options. Do you remember where this was configurable at the time?

If i'm looking in the correct direction i can try to add the option. Else please point me in the right direction to assist on this amazing project.

**Comment by badb** [comment]
Resolved by #1176
